automation-roles/20-prepare/download-cli/tasks/aws-download-cli.yml

```diff
@@ -1,8 +1,11 @@
 ---
-- name: Download ROSA command line tool 
-  include_role:
-    name: aws-download-rosa
+- include_tasks: aws-rosa-download-cli.yml
+  when: _openshift_on_aws_type is defined and _openshift_on_aws_type  == 'rosa'
 
-- name: Download AWS Client
-  include_role:
-    name: aws-download-cli
\ No newline at end of file
+- include_tasks: aws-ipi-download-cli.yml
```

Comment:
There is no need to download the AWS client for every openshift definition. You can download the latest version, only once. I would suggest you always download rosa and aws-cli if AWS.